docs/best-practices/continuous-integration.md

```diff
@@ -207,3 +207,60 @@ What this build plan does is it checks out the source code, then it downloads th
 * Task 2: **Artifact Download**
 * Task 3: **Fastlane**
 
+# Gitlab-CI Integration
+Use [Gitlab-CI Runner](https://gitlab.com/gitlab-org/gitlab-ci-multi-runner) running on an OSX machine
+to build using fastlane.
+
+## Setting up Repository
+
+Add a `.gitlab-ci.yml` file, to trigger fastlane.
+
+```yml
+stages:
+  - unit_tests
```

Comment:
Gitlab-CI would run 2 stages 1) Unit Tests -> and if this finishes successfully it will run 2) Testflight -> you could do it all in one lane, but its supposed to do unittests on every commit, and testflight only on master or release-****